src/webgl/interaction.js

```diff
@@ -88,20 +88,22 @@ p5.prototype.orbitControl = function(sensitivityX, sensitivityY, sensitivityZ) {
     this._setProperty('wheelDefaultDisabled', true);
   }
 
-  const scaleFactor = this.height < this.width ? this.height : this.width;
-
+  const scaleFactor = 50;
   // ZOOM if there is a change in mouseWheelDelta
-  if (this._mouseWheelDeltaY !== this._pmouseWheelDeltaY) {
+  if (this._mouseWheelDeltaY !=0) {
     // zoom according to direction of mouseWheelDeltaY rather than value
     if (this._mouseWheelDeltaY > 0) {
       this._renderer._curCamera._orbit(0, 0, sensitivityZ * scaleFactor);
     } else {
       this._renderer._curCamera._orbit(0, 0, -sensitivityZ * scaleFactor);
     }
+    this._mouseWheelDeltaY=0;
   }
 
   if (this.mouseIsPressed) {
     // ORBIT BEHAVIOR
+    if(Math.abs(this.mouseX-this.pmouseX)>50||Math.abs(this.mouseY-this.pmouseY)>50)
+      return this;//when noloop() is disabled,need a frame to reset pmouse position 
```

Comment:
Makes sense, I see that if I start off with `noLoop()` and then call `loop()` in e.g. `mousePressed`, the orbit jumps. Maybe we can rephrase the comment a bit to make this clearer, something like:
"Ignore abnormally large mouse movements, which might occur if the draw loop has been paused and just became unpaused"